cmd/xfn/main.go

```diff
@@ -69,6 +77,7 @@ func main() {
 		kong.Description("Crossplane Composition Functions."),
 		kong.BindTo(logging.NewLogrLogger(zl), (*logging.Logger)(nil)),
 		kong.UsageOnError(),
+		KongVars,
 	)
-	ctx.FatalIfErrorf(ctx.Run())
+	ctx.FatalIfErrorf(ctx.Run(&start.Args{Registry: cli.Registry}))
```

Comment:
This pattern of passing arguments down to a child command is a little awkward, but AFAICT it's the right way to do this in kong.

I wonder whether we should define the other common flags - e.g. `MapRootUID` - this way too.

Reply:
I thought this was odd too but I couldn't find a better implementation.